tests/acceptance/users_frontend.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given I see the joomla! Home page
+
+  Scenario: Create user from frontend (index.php?option=com_users)
+    Given I click on the link "Create an account"
+    And I create a user with fields Name "patel", Username "patel", Password "patel" and Email "patel@gmail.com"
+    When I press the "Register"
+    Then I see "Could not instantiate mail function." message
+    And user is created
+
+  Scenario: check the created user in the backend
+    Given There is a user manager page in administrator
+    When I search the user with user name "patel"
+    Then I should see the user "prital"
```

Comment:
There is always a user manager - in current Joomla, you can't uninstall it. So it should be

```Gherkin
    Given I am on the User Manager page
```

The reference to the backend is not necessarily needed, because the User Manager page *is* in the backend. But it would not be wrong to say it explicitely.